.github/CODEOWNERS.txt

```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```

Comment:
this file should not contain file extension. It should be named just `CODEOWNERS`

Reply:
I have fixed this extension in the fix commit.